applications/serial_lte_modem/doc/CARRIER_AT_commands.rst

```diff
@@ -98,7 +101,10 @@ The ``<cmd>`` command is a string, and can be used as follows:
 * ``AT#XCARRIER="voltage",<power_source>,<voltage>``
 
 The values of the parameters depend on the command string used.
-When using the ``app_data`` command, if the ``<data>`` attribute is not specified, SLM enters ``slm_data_mode``.
+Specifying ``instance_id`` and ``resource_instance_id`` in the ``app_data`` command sends the data
+to the Binary App Data Container object instead of the App Data Container object.
+When using the ``app_data`` command, if no attributes are specified, SLM enters ``slm_data_mode``.
+``slm_data_mode`` is only supported when using the App Data Container object.
```

Comment:
```suggestion
Specifying ``instance_id`` and ``resource_instance_id`` in the ``app_data`` command sends the data to the Binary App Data Container object instead of the App Data Container object.
When using the ``app_data`` command, if no attributes are specified, SLM enters ``slm_data_mode``.
``slm_data_mode`` is only supported when using the App Data Container object.
```